.gitignore

```diff
@@ -1,10 +1,4 @@
-# Please do not use this ignore file to define platform specific files.
-#
-# For these purposes create a global .gitignore file, which is a list of rules
-# for ignoring files in every Git repository on your computer.
-#
-# https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
-
+.idea
```

Comment:
Please do not use this ignore file to define platform specific files.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
9d1d1fedfa6fcff48cddca97c25e60e93c195518 Fixed